sources/packages/web/src/views/student/StudentApplicationDetails.vue

```diff
@@ -12,6 +12,13 @@
     >
     <Menu class="mt-n15" ref="menu" :model="items" :popup="true" />
     <v-btn
+      v-if="
+        [
```

Comment:
I would prefer to move this logic to a computed property to keep the template with less code.